frontends/concrete-python/tests/dtypes/test_tfhers.py

```diff
@@ -150,3 +172,19 @@ def test_tfhers_encryption_variance(crypto_params: tfhers.CryptoParams):
         return
     assert crypto_params.encryption_key_choice == tfhers.EncryptionKeyChoice.SMALL
     assert crypto_params.encryption_variance() == crypto_params.lwe_noise_distribution**2
+
+
+@pytest.mark.parametrize("params_dict", (DEFAULT_TFHERS_PARAMS_DICT,))
```

Comment:
@andrei-stoian-zama I think we should test some equality here, no?

Reply:
I improved the tests